src/components/page-body/institution/Collaborators.vue

```diff
@@ -169,18 +169,23 @@
             alert('Campos não preenchidos')
             return
           }else if(this.editedFile){ 
-            uploadFile(this.editedFile, `collaborator-${this.editedCollaborator.id}`)
+            const nameFileEdited = `collaborator-${this.editedCollaborator.id}`
+            this.changeImage(nameFileEdited)
+            this.addEditedCollaboratorInCollaborators()
+          } else{
+            this.addEditedCollaboratorInCollaborators()
+          }
```

Comment:
como` this.addEditedCollaboratorInCollaborators()` é sempre chamado, não precisa do `else`

          else if(this.editedFile) { 
            const nameFileEdited = `collaborator-${this.editedCollaborator.id}`
            this.changeImage(nameFileEdited)
          } 
            this.addEditedCollaboratorInCollaborators()

Reply:
Excelente